sql/numbertype.go

```diff
@@ -330,6 +330,39 @@ func (t numberTypeImpl) Convert(v interface{}) (interface{}, error) {
 	}
 }
 
+// MaxResponseByteLength implements the Type interface
+func (t numberTypeImpl) MaxResponseByteLength() uint32 {
+	// https://dev.mysql.com/doc/refman/8.0/en/integer-types.html
```

Comment:
This was a bit confusing at first, I think we should probably state that this is for the lengths of their string forms? Also, wouldn't `Int64` be `21` bytes to account for the additional negative sign?

Reply:
Good feedback. I'll add a comment to call that out, and I'm also going to try renaming the method to `MaxTextResponseByteLength` and see how that looks. I think that might make it more obvious that this is for text serialization. 

Int64 is interesting... it definitely messes up that nice pattern with the other int types, but 20 is what mysql returns. Looking at the [MySQL integer type values ranges](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html), it makes sense though... Max value for int64 is `2^63 - 1` which requires 20 digits in a text representation (including one char for the sign) and for uint64 the max value is `2^64 - 1`, which needs an extra digit for the larger number, but doesn't need the char for the sign, so still works out to be 20.